tools/spectral/ipa/rulesets/IPA-106.yaml

```
@@ -10,24 +10,29 @@ functions:
 
 rules:
   xgen-IPA-106-create-method-request-body-is-request-suffixed-object:
-    description: 'The Create method request should be a Request suffixed object. http://go/ipa/106'
+    description: |
```

Do not use | as it will break the table in markdown
```suggestion
    description: >-
```